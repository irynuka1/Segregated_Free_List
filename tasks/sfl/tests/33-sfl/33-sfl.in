INIT_HEAP 0x1 1 64 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
MALLOC 1
FREE 0x1
FREE 0x2
FREE 0x3
FREE 0x4
FREE 0x5
FREE 0x6
FREE 0x7
FREE 0x8
FREE 0x9
FREE 0xa
FREE 0xb
FREE 0xc
FREE 0xd
FREE 0xe
FREE 0xf
FREE 0x10
FREE 0x11
FREE 0x12
FREE 0x13
FREE 0x14
FREE 0x15
FREE 0x16
FREE 0x17
FREE 0x18
FREE 0x19
FREE 0x1a
FREE 0x1b
FREE 0x1c
FREE 0x1d
FREE 0x1e
FREE 0x1f
FREE 0x20
FREE 0x21
FREE 0x22
FREE 0x23
FREE 0x24
FREE 0x25
FREE 0x26
FREE 0x27
FREE 0x28
FREE 0x29
FREE 0x2a
FREE 0x2b
FREE 0x2c
FREE 0x2d
FREE 0x2e
FREE 0x2f
FREE 0x30
FREE 0x31
FREE 0x32
FREE 0x33
FREE 0x34
FREE 0x35
FREE 0x36
FREE 0x37
FREE 0x38
FREE 0x39
FREE 0x3a
FREE 0x3b
FREE 0x3c
FREE 0x3d
FREE 0x3e
FREE 0x3f
FREE 0x40
DUMP_MEMORY
DESTROY_HEAP
