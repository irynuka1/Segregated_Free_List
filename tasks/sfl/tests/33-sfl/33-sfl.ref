+++++DUMP+++++
Total memory: 64 bytes
Total allocated memory: 0 bytes
Total free memory: 64 bytes
Free blocks: 8
Number of allocated blocks: 0
Number of malloc calls: 64
Number of fragmentations: 56
Number of free calls: 64
Blocks with 8 bytes - 8 free block(s) : 0x1 0x9 0x11 0x19 0x21 0x29 0x31 0x39
Allocated blocks :
-----DUMP-----
