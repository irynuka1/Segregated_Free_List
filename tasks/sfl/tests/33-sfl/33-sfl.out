+++++DUMP+++++
Total memory: 64 bytes
Total allocated memory: 0 bytes
Total free memory: 64 bytes
Free blocks: 64
Number of allocated blocks: 0
Number of malloc calls: 64
Number of fragmentations: 56
Number of free calls: 64
Blocks with 1 bytes - 64 free block(s) : 0x1 0x2 0x3 0x4 0x5 0x6 0x7 0x8 0x9 0xa 0xb 0xc 0xd 0xe 0xf 0x10 0x11 0x12 0x13 0x14 0x15 0x16 0x17 0x18 0x19 0x1a 0x1b 0x1c 0x1d 0x1e 0x1f 0x20 0x21 0x22 0x23 0x24 0x25 0x26 0x27 0x28 0x29 0x2a 0x2b 0x2c 0x2d 0x2e 0x2f 0x30 0x31 0x32 0x33 0x34 0x35 0x36 0x37 0x38 0x39 0x3a 0x3b 0x3c 0x3d 0x3e 0x3f 0x40
Allocated blocks :
-----DUMP-----
